test/edu-benefits/1990e/config/educationHistory.unit.spec.jsx

```diff
@@ -19,7 +19,7 @@ describe('Edu 1990e educationHistory', () => {
           definitions={definitions}/>
     );
     const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
```

Comment:
Does it make more sense to scryRenderedDOMComponentsWithTag rather than getFormDOM here because it is a tag?